### DISCUSSION TITLE
Clear all data

### DISCUSSION THREAD

**Discussion by Moltas18** [Budibase/budibase#6700] [Help] [answered]
Hi! I've tried out Budibase for a while now and I'm very much enjoying it. I have built an Application which I intend to deploy locally. The application is a tool for projects. When a project is completed and I want to start a new, I would like to Deploy the application repeatedly, but empty, without data in the tables and with reset Auto ID. My plan is to export a copy of the application that I've built that is empty. When I tried this though, the Auto ID's weren't reset. Is there a way to clear all the data in a Budibase Application so that everything is reset?

**Comment by gerhardmpl** [comment, marked as answer]
@Moltas18 I think there is no good way to do so yet (see #4630)

Edit: Maybe a set of forEach `delete row` automations triggered in an Admin screen can at least clear the tables in a defined sequence (but not reset the Auto ID)? Mh, probably a bad idea due to binding limitations...

**Reply by Moltas18** [reply]
I see. I read #4630. My problem is not the efficiency though. I don't have large sets of data that I need to delete, rather just reset everything so that when I launch a new project, all rows begin from scratch. I might actually try to export the whole application and delete the data from the "source code". Thanks anyway for the input!

**Reply by Moltas18** [reply]
Worked out a way. Not the most user friendly but it did the job. And since I can copy it empty, there's no need to do it more than once when the application is finished.

I accessed the CocuchDB database via http://localhost:10000/db/_utils/#login and changed "lastID": 1 -> "lastID": 0 within the table documents.
